### ISSUE TITLE
Reset IRS environment and testdata import as github action

### ISSUE THREAD

**Issue by ds-alexander-bulgakov** [eclipse-tractusx/item-relationship-service#255]
**As** team member,
**I want** a github action to reset an environment,
**so that** no manual actions are necessary anymore.

## Hints / Details
- TRACEFOSS has already an action which is hard resetting the desired env, delete env, sync all necessary environments and upload data. Branch and testdata can be adjusted before starting the job.

### Outcome
- Github action which can be used to completely reset desired environment with desired branch and testdata automatically.
### Acceptance Criteria
- Github action can reset the environment, set up all necessary environments and upload desired testdata

## Out of Scope
- ...

**Comment by jzbmw** [comment]
closed since these efforts are not neccessary for rest time period of consortia
